setup.py

```diff
@@ -114,6 +114,11 @@ def load_version_module(pkg_path):
         'k8s': [
           'kubernetes',
         ],
+
+        # For including XProf server
```

Comment:
Do we need this extra? I feel like users can just install xprof themselves (that's what the docs say)